.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: android
+
+jdk: oraclejdk8
+
+android:
+  components:
+    - tools
+    - android-25
+    - build-tools-25.0.1
+    - platform-tools
+    - extra-android-m2repository
+
+after_failure:
+    - cat $TRAVIS_BUILD_DIR/viewpump/build/reports/lint-results.xml || true
```

Comment:
oh that's cool! guess i would've known that if i used gradle `3.4.1-all` :smiley: